src/PresentationalComponents/DeniedState/DeniedState.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card } from '@patternfly/react-core/dist/js/components/Card/Card';
+import { Title } from '@patternfly/react-core/dist/js/components/Title/Title';
+import {
+    EmptyState,
+    EmptyStateVariant,
+    EmptyStateIcon,
+    EmptyStateBody
+} from '@patternfly/react-core/dist/js/components/EmptyState';
+
+import { LockIcon } from '@patternfly/react-icons/dist/js/icons/lock-icon';
```

Comment:
this throws 
```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: null.

Check the render method of `EmptyStateIcon`.

```

gotta do the import like this: `import LockIcon  from '@patternfly/react-icons/dist/js/icons/lock-icon';`